object_to_location_max_distance: 1.75